pyincore/analyses/bridgedamage/bridgedamage.py

```diff
@@ -328,91 +328,107 @@ def get_spec(self):
         """
         return {
             "name": "bridge-damage",
-            "description": "bridge damage analysis",
+            "description": (
+                "This analysis computes bridge damage based on a particular hazard. Currently "
+                "supported hazards are: earthquake, tsunami, tornado, and hurricane.\n"
+                "The process for computing the structural damage is similar to other parts of "
+                "the built environment. First, a fragility is obtained based on the hazard type "
+                "and attributes of the bridge. Based on the fragility, the hazard intensity at "
+                "the location of the bridge is computed. Using this information, the probability "
+                "of exceeding each limit state is computed, along with the probability of damage.\n"
+                "For the case of an earthquake hazard, if the bridge dataset contains soil "
+                "information, the median value of the associated fragility can be modified to "
+                "account for liquefaction in the damage.\n"
+                "The outputs of this analysis are a CSV file with probabilities of damage and "
+                "a JSON file with information about hazards and fragilities."
+            ),
             "input_parameters": [
                 {
                     "id": "result_name",
                     "required": True,
-                    "description": "result dataset name",
+                    "description": "Set custom result dataset name.",
```

Comment:
Not sure what you all think, but for this I was changing it for all analyses to say:
"Base name of the result output."